rpcserver.go

```diff
@@ -4104,7 +4104,6 @@ func (r *rpcServer) AddInvoice(ctx context.Context,
 		IsChannelActive:   r.server.htlcSwitch.HasActiveLink,
 		ChainParams:       activeNetParams.Params,
 		NodeSigner:        r.server.nodeSigner,
-		MaxPaymentMSat:    MaxPaymentMSat,
```

Comment:
We still use it at times to ensure that routes submitted don't have hops which attempt to send more than that value.

Reply:
Why would we allow a simple `payinvoice` to send out an htlc of any size, and restrict what gets offered via `SendToRoute`?

Reply:
Assuming `payInvoice` does payment splitting then it would/should adhere to the max split size of 2.4 mil satoshis. On the other hand, nodes should/will reject an HTLC above that size, so all routes need to always be below that value.